tools/hsettp/kvs-cn-tree-get.c

```diff
@@ -21,7 +21,7 @@
 #pragma GCC visibility push(default)
 
 const char *kvs_cn_tree_get_headers[] = {
-    "T",
+    "H",
```

Comment:
What does H mean?

Reply:
Header?  Changed to match what cn_metrics.c produces.

Reply:
T (type) makes more sense than H (header) imo

Reply:
Agreed, I could change it in cn_metrics, I doubt anyone is keying on it..

Reply:
I would double check with @rahulreddy7

Reply:
We aren't using that option in SQA.

Reply:
i've changed it to 'T' in cn_metrics.